src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -166,7 +166,7 @@ def _add_optional_columns(df: pd.DataFrame) -> pd.DataFrame:
     in_df_cols = set(df.columns)
     if not optional_col_set.issubset(in_df_cols):
         additional_col = list(optional_col_set.difference(in_df_cols))
-        additional_df = pd.DataFrame(columns=additional_col, index=df.index, data=None)
+        additional_df = pd.DataFrame(columns=additional_col, index=df.index, data=pd.NA)  # type: ignore[call-overload]
```

Comment:
If this is not `pd.NA` (`np.nan` is likely to work also) then it is not recognised as empty. This causes a problem in the code in utils.

Reply:
I'm not enough knowledgeable to complain, but I assume that the typing system knows better than we do. If we introduce something that the type system doesn't like, it should make us suspicious. if So perhaps it's worth digging deeper here, to make sure that we don't introduce another bug.
But if you're confident enough, I don't want to block you.

Reply:
So do you think there is a bug in the pandas stubs? Because the `data=pd.NA` is not accepted by the stubs, which makes me suspicious.

Reply:
What was Daniela's problem? Could it be solved in another way?

Reply:
Honestly, I don't really understand this either. Because if you print out `pd.isna(None)` you get `True`. But pandas really is super difficult with nan values. That I can say for sure. Maybe the problem was that the nan values in the df were now mixed, because while they both are nan they are not identical, see screenshot. But giving `data=None` or `pd.NA` seem like it is a bad idea so, at least we fixed something we understand. I checked this cannot be found anywhere else in the code.

<img width="1058" alt="Screenshot 2024-04-09 at 10 58 05" src="https://github.com/dasch-swiss/dsp-tools/assets/103038637/332fc638-8757-4462-aaac-07c6b1cae92f">